__tests__/app.test.js

```diff
@@ -202,4 +202,191 @@ describe("api/articles/:article_id/comments", () => {
+    .send( {body: "this is a test"} )
+    .expect(400)
+    .then( data => {
+      expect(data.body.msg).toBe("incomplete request, provide username")
+   })
+  })
+
+  test("responds with 400 and error if body has not been defined", () => {
+
+    return request(app)
+    .post("/api/articles/5/comments")
+    .send( {username: "icellusedkars"} )
+    .expect(400)
+    .then( data => {
+      expect(data.body.msg).toBe("incomplete request, please add a comment")
+   })
+  })
+
+}) // end of "POST /api/articles/:article_id/comments" testing
+
+describe("PATCH: /api/articles/:article_id", () => {
+
+  test("returns 202 status code following successful request", () => {
+
+   return request(app)
+    .patch("/api/articles/5")
+    .send( { inc_votes: 35 } )
+    .expect(202)
+  })
+
+  test("returns article with updated number of votes", () => {
+    return request(app)
+      .patch("/api/articles/9")
+      .send( { inc_votes: 15 } )
+      .then( data => {
+        expect(data.body.article).toMatchObject({
+          article_id: 9,
+          title: "They're not exactly dogs, are they?",
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'Well? Think about it.',
+          created_at: "2020-06-06T09:10:00.000Z",
+          votes: 15,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        })
+      })
+  })
+
+  test("votes should still update when passed a negative number", () => {
+    return request(app)
+      .patch("/api/articles/9")
+      .send( { inc_votes: -5 } )
+      .then( data => {
+        expect(data.body.article).toMatchObject({
+          article_id: 9,
+          title: "They're not exactly dogs, are they?",
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'Well? Think about it.',
+          created_at: "2020-06-06T09:10:00.000Z",
+          votes: -5,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        })
+      })
+  })
+
+  test("returns 400 and error when no votes object passed", () => {
+    return request(app)
+      .patch("/api/articles/9")
+      .send()
+      .expect(400)
+      .then( data => {
+        expect(data.body.msg).toBe("no votes submitted")
+      })
+  })
+
+  test("return 404 and error if unable to match article_id", () => {
+    return request(app)
+      .patch("/api/articles/9999")
+      .send( { inc_votes: 25 } )
+      .expect(404)
+      .then( data => {
+      expect(data.body.msg).toBe("no article found");
+    })
+  })
+
+}) // end of "PATCH: /api/articles/:article_id" testing
+
+describe("DELETE: /api/comments/:comment_id", () => {
+
+  test("responds with 204 status, deletes the comment object", () => {
+   return request(app)
+    .delete("/api/comments/14")
+    .expect(204)
+    .then( () => {
+      return request(app).get("/api/articles/5/comments")
```

Comment:
I like your thinking here! But I think this test would be fine without the get request, as the status 204 is being showing that the comment has been deleted